reports/usage-analytics/index.md

```diff
@@ -60,13 +60,16 @@ First create a custom object. To follow along with this example name it _Skuid_U
   - If you created a custom object in Salesforce, most of the usage tracking functionality can leverage default fields on a custom object, like _Last Activity Date_ and _Owner_. 
   - If you want to avoid creating extra fields, you can use the standard _Name_ field on the custom object and rename it.
 
-**With the object created, let's adjust the object's fields.** This example uses the following standard fields to gather basic information:
+With the object created, let's adjust the object's **Name field,** which we'll use as a **text** field to capture the name of the Skuid page where the tracked action is taking place. For readability, update this field to be named **Page Name.**
 
-1. **Name**: A **text** field to capture the name of the Skuid page where the tracked action is taking place. For readability, update this field to be named **Page Name.**
-2. **CreatedById**: A **lookup relationship** field that will reference the User object, tying the Skuid Usage record to the user that performed the tracked action.
-3. **Created Date**: A **date/time** field that will capture a timestamp of when the Skuid Usage record was created.
+The following two fields, which are automatically populated by Salesforce, will also be useful for analytics tracking:
 
-With those fields set, [create the following fields](https://help.salesforce.com/articleView?id=adding_fields.htm&type=5) to track additional information:
+1. **CreatedById**: A **lookup relationship** field that will reference the User object, tying the Skuid Usage record to the user that performed the tracked action.
+1. **Created Date**: A **date/time** field that will capture a timestamp of when the Skuid Usage record was created.
```

Comment:
```suggestion
2. **Created Date**: A **date/time** field that will capture a timestamp of when the Skuid Usage record was created.
```

Reply:
Markdown will render numbered lists in the correct order despite what the actual number is... But we use actual numbering throughout, yeah let's be consistent.